config/services.yaml

```diff
@@ -47,3 +47,9 @@ services:
 
     App\Content\Type\AlbumSelection:
         tags: [{name: 'sulu.content.type', alias: 'album_selection'}]
+
+    App\Content\Type\SingleEventSelection:
+        tags: [ { name: 'sulu.content.type', alias: 'single_event_selection' } ]
+
+    App\Content\Type\EventSelection:
+        tags: [ { name: 'sulu.content.type', alias: 'event_selection' } ]
```

Comment:
If a field-type is used inside of a page template, **Sulu looks for a content-type that matches the name of the field-type**. A content-type is registered as Symfony service and is responsible for loading the data that is passed to the Twig template of the page.

If a field-type is not used inside of page templates, it is not necessary to implement a matching content-type.